validation/views.py

```diff
@@ -928,3 +942,26 @@ def create_new_rec_id(phrase, speaker):
         f"{time.time()}\n"
     )
     return sha256(signature.encode("UTF-8")).hexdigest()
+
+
+def save_metadata_to_file(rec_id, user, transcription, translation):
+    dest = (
+        settings.MEDIA_ROOT
+        + "/"
+        + settings.RECVAL_AUDIO_PREFIX
+        + "metadata/"
+        + rec_id
+        + ".json"
+    )
+    data = {
+        "audio_file_name": rec_id + ".wav",
+        "user_id": user.id,
+        "username": user.username,
+        "full_name": user.first_name + " " + user.last_name,
+        "recorded_on": str(datetime.datetime.now()),
```

Comment:
Sadly this leaves out the timezone information. I had to look up a way to include that:

```suggestion
        "recorded_on": str(datetime.datetime.now().astimezone()),
```